operations/rates.md

```diff
@@ -169,7 +251,7 @@ The price in the response is dependent on the enterprise's [pricing](configurati
 | :-- | :-- | :-- | :-- |
 | `Currency` | string | required | ISO-4217 code of the [Currency](currencies.md#currency). |
 | `TimeUnitStartsUtc` | array of string | required | Set of all time units covered by the time interval; expressed in UTC timezone ISO 8601 format. |
-| `BasePrices` | array of number | required | Base prices of the rates for each time unit covered by the time interval. |
+| `BaseAmountPrices` | [Amount value](accountingitems.md#amount-value) | required | Base prices of the rates for each time unit covered by the time interval. |
```

Comment:
This points to an object definition for `Amount value` in accountingitems.md, however if I look at that object definition it is missing "Breakdown" so it's different to the usage in the example JSON here.

Reply:
I know about this and wanted to resolve in separate issue. The object definition for `Amount value` in `accountingitems.md` is outdated for a long time since it has more values that just Breakdown that is missing. Somebody forgot to update it about a year ago.

+ these objects for me indicates we should unify these objects that are reused across multiple endpoints since it makes it harder to keep it up to date (not in this case but I would avoid creating duplications but it's up to you.

Just FYI we have two objects for Amount in the code

**Amount:**
```
 {
      "Currency": "EUR",
      "NetValue": 93.46,
      "GrossValue": 100.00,
      "TaxValues": [
          {
              "Code": "DE-2020-1-L",
              "Value": 6.54
          }
      ],
      "Breakdown": {
          "Items": [
              {
                  "TaxRateCode": "DE-2020-1-L",
                  "NetValue": 93.46,
                  "TaxValue": 6.54
              }
          ]
      }
}
```

**ExtendedAmount** (the values are incorrect, I'm just showing it as an handmade example).
```
 {
      "Currency": "EUR",
      "Value" : "",
      "NetValue": 93.46,
      "Net" : "",
      "GrossValue": 100.00,
      "Tax" : "",
      "TaxRate" : "",
      "TaxValues": [
          {
              "Code": "DE-2020-1-L",
              "Value": 6.54
          }
      ],
      "Breakdown": {
          "Items": [
              {
                  "TaxRateCode": "DE-2020-1-L",
                  "NetValue": 93.46,
                  "TaxValue": 6.54
              }
          ]
      }
}
```
PS. I have no idea about reason behind it.

Reply:
Oh wow, thanks for the info, I didn't know that. Well yes, whatever the reason, the current documentation is in error and should be corrected, so yes please if you could update it that would be great. That's fine by me if it's a separate PR, but there should be no delay because this PR is technically not correct otherwise. In terms of common object definitions that are used in multiple endpoints, the policy is already that they are defined in one place, hence you are referencing `Amount Value` on the `Accounting Items` page. There is actually a statement about this in the [Style Guide](https://app.getguru.com/card/c98GRexi/Style-Guide-for-Mews-Open-API-Documentation).

Reply:
@MikeAdamsMews How is it now it's correct, it looks like in documentation is without deprecated ones (Net, Tax, TaxRates are depercated), so how is it now is correct.

Reply:
Not quite sure I follow what you are saying, the point is simply to make sure that the example JSON matches the definition of `Amount value`. As long as it does, I'm happy :-)

Reply:
It corresponds to the code :) Deprecated values are not there so it's ok now :)